packages/schemas/src/seeds/application.ts

```diff
@@ -0,0 +1,18 @@
+import { ApplicationType, CreateApplication } from '../db-entries';
+
+/**
+ * The fixed application ID for Admin Console.
+ *
+ * This built-in application does not belong to any tenant in the OSS version.
+ */
+export const adminConsoleApplicationId = 'admin_console';
+
+export const demoAppApplicationId = 'demo_app';
+
+export const createDemoAppApplication = (urls: string[]): Readonly<CreateApplication> => ({
```

Comment:
App and Application together look redundant.

Reply:
`demo app` is the name

Reply:
Can't we just use `demoApplicationId` and `createDemoApplication()`?

Reply:
then we need to change all the occurrences across the project: route, phrase, etc. (demo app -> demo)

Reply:
Well, OK. Will take `demo app` as a normal application name as `admin console`

Reply:
cool